applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_check_utilities.cpp

```diff
@@ -46,4 +67,45 @@ KRATOS_TEST_CASE_IN_SUITE(CheckUtilities_CheckDomainSize, KratosGeoMechanicsFast
         "DomainSize (0) is smaller than 1e-15 for element 1")
 }
 
+KRATOS_TEST_CASE_IN_SUITE(CheckUtilities_CheckNodalVariables, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Arrange
+    auto line_geometry = CreatLineGeometryWithoutVariables();
+
+    // Act and Assert
+    KRATOS_EXPECT_EXCEPTION_IS_THROWN(
+        CheckUtilities::CheckNodalVariables(line_geometry, {WATER_PRESSURE, VOLUME_ACCELERATION}),
```

Comment:
If you follow my earlier suggestion of using `Geo::ConstVariableDataRefs`, creating the vector of variable references needs to be done as follows:
```suggestion
        CheckUtilities::CheckNodalVariables(line_geometry, {std::cref(WATER_PRESSURE), std::cref(VOLUME_ACCELERATION)}),
```